src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/util/PortWaiter.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.util;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.net.Socket;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.awaitility.Awaitility.await;
+
+public final class PortWaiter {
```

Comment:
Probably we could also get rid of this port waiter class and use dynamic port ?

Reply:
yes, this becomes retired after 4th PR located in `refactor/integration-tests-ccd` branch. However, to not mix things up - should do a sanity clean-up after all 6 PRs have been reviewed